images/win/scripts/Installers/Validate-Msys2.ps1

```diff
@@ -0,0 +1,57 @@
+################################################################################
+##  File:  Validate-Msys2.ps1
+##  Desc:  Validate Msys2
+################################################################################
+
+if (Get-Command -Name 'gcc')
+{
+    Write-Host "gcc is successfully installed:"
+    gcc --version | Write-Host
+}
+else
+{
+    Write-Host "gcc is not on PATH"
+    exit 1
+}
+
+if (Get-Command -Name 'g++')
+{
+    Write-Host "g++ is successfully installed:"
+    g++ --version | Write-Host
+}
+else
+{
+    Write-Host "g++ is not on PATH"
+    exit 1
+}
+
+if (Get-Command -Name 'make')
```

Comment:
Will this PR affect default tools somehow, like `tar` and etc. 
As I remember, @miketimofeev had some problems with it previously.

Reply:
We should definitely check all the utilities like tar, perl, and openssl. All of them are supposed to be not from msys or mingw.

Reply:
Re path issues/conflicts, see:
https://github.com/MSP-Greg/actions-image-testing/runs/524240890?check_suite_focus=true#step:11:4